src/sos/workflow_executor.py

```diff
@@ -1646,7 +1646,7 @@ def run_as_master(self, targets=None, mode=None) -> Dict[str, Any]:
                             f'{len(sections)} pending step{"s" if len(sections) > 1 else ""}: {", ".join(sections)}'
                         ),
                     )
-                    raise exec_error
+                    raise error
```

Comment:
No. exec_error is not the same exception as raised from `except ... ` Please revert back.

Reply:
I just checked the latest commit. That error has been resolved now.

Reply:
double check please. I still see `raise error`.